inc/Styles/Component.php

```diff
@@ -85,6 +85,18 @@ public function action_enqueue_styles() {
 			wp_enqueue_style( 'wp-rig-fonts', $google_fonts_url, array(), null ); // phpcs:ignore WordPress.WP.EnqueuedResourceParameters.MissingVersion
 		}
 
+		// Enqueue parent theme style if this is a child theme.
+		if ( is_child_theme() ) {
```

Comment:
2 thoughts:

1) Isn't common practice to @import via the child theme stylesheet?
2) But more importantly: we don't have any CSS in our style.css anymore in v2.0?